demo/src/examples/Tooltip.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { Flex } from '@ivoryio/kogaio'
+import { Tooltip } from '../../../src'
+
+const TooltipExamples = props => (
+  <Flex>
+    <Tooltip {...props} />
```

Comment:
In demo try to _call_ the component as a coder would call it. For instance `Tooltip` is meant to be used as:
```jsx
<Tooltip>This is a sample tooltip</Tooltip>
```
Right now the interface for `Tooltip` is obscured by `{...props}`. 

In other words we should strive for the code under `demo/` to be copy-paste-able. If a user copies a snippet of code from here and pastes it in his app, it should work.